src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -62,10 +63,51 @@ class BaseSidebarScreen extends Component {
         Welcome.show({routes, showCreateMenu: this.showCreateMenu});
     }
 
+    componentDidUpdate(prevProps) {
+        if (!this.didBecomeInactive(prevProps)) {
+            return;
+        }
+
+        // Hide menu manually when other pages are opened using shortcut key
+        this.hideCreateMenu();
+    }
+
+    didBecomeInactive(prevProps) {
+        // When Report page is just opened
+        if (!this.props.isDrawerOpen && prevProps.isDrawerOpen) {
+            return true;
+        }
+
+        // When other page is just opened
+        if (!this.props.isFocused && prevProps.isFocused) {
+            return true;
+        }
+
+        return false;
+    }
+
+    isInactive() {
+        // When drawer is closed and Report page is open
+        if (this.props.isSmallScreenWidth && !this.props.isDrawerOpen) {
+            return true;
+        }
+
+        // When other page is open
```

Comment:
This returns true if any other page is open? Or a specific page? Let's be a bit clearer here

Reply:
Cool so how about this

```suggestion
        // When any other page is open
```